### ISSUE TITLE
Some markdown not rendered

### ISSUE THREAD

**Issue by joho1968** [julien-nc/welcome#31]
Using NC25, I added some text, marked it and then chose the "info callout" formatting.

This produces the following markdown:
`::: info Bla Bla Bla`

Which is rendered as plain text in the dashboard widget.

**Comment by julien-nc** [comment]
Hi,
We use the RichText component from `@nextcloud/vue-richtext` to render the markdown file in the dashboard widget. This component unfortunately does not support callouts.

This could be solved by using the editor package exposed by the Text app like it's done for the Deck card comments for example. This would bring the exact same level of markdown support than in Text.
I don't have time to implement this but I'm very open to pull requests.

**Comment by poVoq** [comment]
Checkboxes `- [ ]` are also not rendered right now.

**Comment by joho1968** [comment]
Yeah, I'd be happy to take a crack at fixing this, but my first PR (#34) created on the 24th of March still hasn't been even looked at it seems. I'm wondering if a fork of this app would be a better idea. I really don't want to step on anyone's toes, so if this is just a temporary "issue", then I'll keep waiting 😊
